dags/vz_socrata_export.py

```diff
@@ -0,0 +1,72 @@
+import os
+from pendulum import datetime, duration
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from utils.slack_operator import task_fail_slack_alert
+from utils.onepassword import get_env_vars_task
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Downloads CR3 pdfs from CRIS and uploads to S3",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "retries": 0,
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+REQUIRED_SECRETS = {
+    "HASURA_ENDPOINT": {
+        "opitem": "Vision Zero CRIS Import",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.GraphQL Endpoint",
+    },
+    "HASURA_ADMIN_KEY": {
+        "opitem": "Vision Zero CRIS Import",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.GraphQL Endpoint key",
+    },
+    "SOCRATA_API_KEY_ID": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "socrata.apiKeyId",
+    },
+    "SOCRATA_API_KEY_SECRET": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "socrata.apiKeySecret",
+    },
+    "SOCRATA_APP_TOKEN": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "socrata.appToken",
+    },
+    "SOCRATA_DATASET_CRASHES": {
+        "opitem": "Vision Zero Socrata Export",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.SOCRATA_DATASET_CRASHES",
+    },
+    "SOCRATA_DATASET_PERSONS": {
+        "opitem": "Vision Zero Socrata Export",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.SOCRATA_DATASET_PERSONS",
+    },
+}
+
+with DAG(
+    dag_id=f"vz_socrata_export_{DEPLOYMENT_ENVIRONMENT}",
+    default_args=default_args,
+    schedule_interval="0 9 * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
```

Comment:
it looks like this is running at 9am UTC on airflow v1, so i think this needs to be adjusted to CDT.

Reply:
🙏 Will update this